test/fp_to_int_intrinsic.ll

```diff
@@ -0,0 +1,236 @@
+;; Ensure @llvm.fptosi.sat.* and @llvm.fptoui.sat.* intrinsics are translated
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
```

Comment:
We should be able to add a `spirv-val` step here, to verify that the generated SPIR-V is valid.

Reply:
Test moved to llvm-intrinsic directory.  Added spirv-val check.